sp/src/vpc_scripts/source_base.vpc

```diff
@@ -21,6 +21,9 @@ $Conditional MAPBASE	"1"
 
 // Toggles Mapbase's RPC implementation
 $Conditional MAPBASE_RPC	"1"
+
+// Toggles VScript implementation (note: interfaces still exist, just the provided implementation is not present)
+$Conditional MAPBASE_VSCRIPT	"1"
```

Comment:
Thank you for adding a unique VPC conditional and C++ preprocessor for this. I will probably use it for any Mapbase-specific expansions or changes to the VScript implementation unless you intended it to be used for more specific purposes only. (e.g. how it's used in `vscript_shared.cpp`)